docs/standard/io/file-path-formats.md

```diff
@@ -0,0 +1,171 @@
+
+> [!IMPORTANT]
+> Relative paths are dangerous in multithreaded applications (that is, most applications) because the current directory is a per-process setting. Any thread can change the current directory at any time. 
+
+### Canonicalizing separators
+
+All forward slashes (`/`) are converted into the standard Windows separator, the back slash (`\`). If they are present, series of slashes that follow the first two slashes are collapsed into a single slash.
+
+> [!IMPORTANT]
+> It is only when identifying paths for normalization purposes that the direction of the slash does not matter. Forward slashes are not supported in Windows outside of this normalization step. This is critically important if normalization is skipped.
+
+### Evaluating relative components
+
+As the path is processed, any components or segments that are composed of a single or a double period (`.` or `..`) are evaluated: 
+
+- For a single period, the current segment is removed, since it refers to the current directory).
+
+- For a double period, the current segment and the parent segment are removed, since the double period refers to the parent directory.
+
+   Parent directories are only removed if they aren't past the root of the path. The root of the path depends on the type of path. It is the drive (`C:\`) for DOS paths, the server/share for UNCs (`\\Server\Share`), and the device path prefix for device paths (`\\?\` or `\\.\`).
+
+### Trimming characters
+
+Along with the runs of separators and relative segments removed earlier, some additional characters are removed during normalization:
+
+- If a segment ends in a single period, that period is removed. (A segment of a single or double period is normalized in the previous step. A segment of three or more periods is not normalized and is actually a valid file/directory name.
+
+- If the path doesn't end in a separator, all trailing periods and spaces (U+0020) are removed. If the last segment is simply a single or double period, it falls under the relative components rule above. 
+
+   This rule leads to the possibly surprising ability to create a directory with a trailing space. You simply need to add a trailing separator to do so.
+
+## Skipping normalization
+
+Normally, any path passed to a Windows API is (effectively) passed to the [GetFullPathName function](https://msdn.microsoft.com/library/windows/desktop/aa364963(v=vs.85).aspx) and normalized. There is one important exception: a device path that begins with a question mark instead of a period. Unless the path starts exactly with `\\?\` (note the use of the canonical backslash), it is normalized.
+
+Why would you want to skip normalization? There are three major reasons:
+
+1. To get access to paths that are normally unavailable but are legal. A file or directory called `hidden.`, for example, is impossible to access in any other way. 
```

Comment:
Nit: I generally prefer using "1." for the start of each bullet so I don't have to re-number if I add or remove steps.

Reply:
@BillWagner, actually so do I, but I've fallen out of the practice of doing it. Thanks for reminding me.